build/index.js

```diff
@@ -386,6 +386,10 @@ var matchLeave = function (ctx, logger, nk, dispatcher, tick, state, presences)
     if (playersRemaining.length === 1) {
         dispatcher.broadcastMessage(OpCode.OPPONENT_LEFT, null, playersRemaining, null, true);
     }
+    else if (state.ai && playersRemaining.length === 0) {
```

Comment:
didn't catch this originally but minor suggested rename to `humansRemaining`, but not blocking approval ofc